packages/blocks/microsoft-outlook/src/lib/actions/reply-email.ts

```diff
@@ -116,11 +116,15 @@ export const replyEmailAction = createAction({
       file: WorkflowFile;
       fileName: string;
     }>;
+
+    const formattedReplyBody =
```

Comment:
so, the thing is outlook either allows body OR comment, its not possible to have both see -> https://learn.microsoft.com/en-us/graph/api/message-reply?view=graph-rest-1.0&tabs=http 

"Specify either a comment or the body property of the message parameter. Specifying both will return an HTTP 400 Bad Request error."

so this is my solution to still keep body format in logically, wdyt?